### ISSUE TITLE
strings are hard to understand in XMIR, without comments

### ISSUE THREAD

**Issue by yegor256** [objectionary/jeo-maven-plugin#847]
Currently, I see this in XMIR:

```xml
<o base="org.eolang.jeo.string">
  <o base="bytes" data="bytes">6A 61 76 61 2F 6C 61 6E 67 2F 4F 62 6A 65 63 74</o>
</o>
```

Would be much more readable, if we would have this instead:

```xml
<o base="org.eolang.jeo.string">
  <!-- "java/lang/Object" -->
  <o base="bytes" data="bytes">6A 61 76 61 2F 6C 61 6E 67 2F 4F 62 6A 65 63 74</o>
</o>
```

Similar comments would be good to have for `org.eolang.jeo.int` types too (and others).

**Comment by yegor256** [comment]
@volodya-lombrozo please, check. With these comments, the XMIR would look much more readable.

**Comment by volodya-lombrozo** [comment]
@yegor256 This issue was solved in the `0.6.19` version as well. Can we close it? Similar to https://github.com/objectionary/jeo-maven-plugin/issues/849
